src/api/adminApi.tsx

```diff
@@ -82,6 +82,12 @@ export const getUsersByExactRoles = async (
   return response;
 };
 
+export const getUsersForGoverningBodies = async () => {
+  const response = await Api.get(`Admin/GetUsersForGoverningBodies`);
+
+  return response;
```

Comment:
I think this looks better
```suggestion
  return await Api.get(`Admin/GetUsersForGoverningBodies`);
```
Also, what if an error occurs?
You should add `catch` just in case.

Reply:
Thank, fixed